client/landing/stepper/declarative-flow/internals/steps-repository/domain-contact-info/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { camelToSnakeCase, mapRecordKeysRecursively, snakeToCamelCase } from '@automattic/js-utils';
+import { CheckboxControl } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { StepContainer } from 'calypso/../packages/onboarding/src';
+import ContactDetailsFormFields from 'calypso/components/domains/contact-details-form-fields';
+import FormattedHeader from 'calypso/components/formatted-header';
+import useDomainTransferReceive from 'calypso/data/domains/transfers/use-domain-transfer-receive';
+import { useDomainParams } from 'calypso/landing/stepper/hooks/use-domain-params';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import wp from 'calypso/lib/wp';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import type { StepProps, ProvidedDependencies } from '../../types';
+import './styles.scss';
+
+export default function DomainContactInfo( { navigation }: StepProps ) {
+	const { submit } = navigation;
+	const { __ } = useI18n();
+
+	return (
+		<StepContainer
+			hideBack
+			stepName="domain-contact-info"
+			isLargeSkipLayout={ false }
+			formattedHeader={
+				<FormattedHeader
+					id="domain-contact-info__header"
+					headerText={ __( 'Enter your contact informaiton' ) }
```

Comment:
@lsl Just noting there's a typo here. 

```suggestion
					headerText={ __( 'Enter your contact information' ) }
```